apps/docs/.vitepress/config.ts

```diff
@@ -324,6 +325,7 @@ export default defineConfigWithTheme<ThemeConfigExtended>({
       stringify: true,
     },
     plugins: [
+      SearchPlugin(),
```

Comment:
what plugins do require this add-on? 🕵🏼

Reply:
Moved from deleted vite.config.ts file :scream_cat: